### DISCUSSION TITLE
How to make insertNode with default value from function

### DISCUSSION THREAD

**Discussion by geril07** [L3MON4D3/LuaSnip#961] [Q&A] [answered]
I want to make behaviour where insertNode's default value would have  part of filename. I looked at functionNode, but it does not focusable. Maybe there is some option for this or i need other node?

**Comment by L3MON4D3** [comment, marked as answer]
Yeah, check out `dynamicNode`, it can generate `insertNode`s

**Reply by geril07** [reply]
Yeah, i found solution with dynamicNode, thank you.
